Prace_domowe/Praca_domowa3/Grupa1/Sebastian_Deregowski/PD3_Deregowski_Sebastian.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>Wyprzedził Pan treści prezentowane na laboratorium, <strong>ale to super, że ma Pan dużą wiedzę z ML i stosuje Pan bardziej zaawansowane techniki nawet gdy nie są one wymagane na danym etapie.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>